.env

```diff
@@ -1,4 +1,5 @@
 NODE_ENV=development
 PORT = 5000
 jwtPrivateKey="mybroadcasterapp"
-test_token="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNTc0NTE0Njc5fQ.oGR9BysE4MCTX4ONRLZnqUirnGNIJXJGtaWD7b-4M0U"
\ No newline at end of file
+test_token="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNTc0NTE0Njc5fQ.oGR9BysE4MCTX4ONRLZnqUirnGNIJXJGtaWD7b-4M0U"
+DATABASE_URL = 'postgresql://postgres:4512@localhost:/broadcaster'
```

Comment:
In this way, It means you have exposed your database.
Please add `.env` to the `.gitignore`

Reply:
done